Purchases.podspec

```diff
@@ -22,7 +22,7 @@ Pod::Spec.new do |s|
   s.tvos.deployment_target = '9.0'
 
   s.pod_target_xcconfig = { 'DEFINES_MODULE' => 'YES' }
-  s.dependency 'PurchasesCoreSwift', '3.6.0'
+  s.dependency 'PurchasesCoreSwift'
```

Comment:
Is this going to resolve to the latest available? If that's the case, that could break for people using older versions right?

Reply:
it resolves to the latest available, but that should be fine for the `develop` branch, right? we need to make sure that on every release, it gets tied to a specific version. 
but it's probably a good idea to keep it tied together, I'll update to 3.7.0-SNAPSHOT